docs/docusaurus/docs/cloud/expectations/manage_expectations.md

```diff
@@ -16,52 +16,53 @@ An Expectation is a verifiable assertion about your data. They make implicit ass
-| Schema             | `expect_table_columns_to_match_set`                       | Expect the columns in a table to match an unordered set.                                                                               | No |
-| Sets               | `expect_column_values_to_be_in_set`                       | Expect each column value to be in a given set.                                                                                         | No |
-| Sets               | `expect_column_values_to_not_be_in_set`                   | Expect column entries to not be in the set.                                                                                            | No |
-| Sets               | `expect_column_distinct_values_to_be_in_set`              | Expect the set of distinct column values to be contained by a given set.                                                               | No |
-| Sets               | `expect_column_distinct_values_to_contain_set`            | Expect the set of distinct column values to contain a given set.                                                                       | No |
-| Sets               | `expect_column_distinct_values_to_equal_set`              | Expect the set of distinct column values to equal a given set.                                                                         | No |
-| Sets               | `expect_column_most_common_value_to_be_in_set`            | Expect the most common value to be within the designated value set.                                                                    | No |
-| Volume             | `expect_table_row_count_to_be_between`                    | Expect the number of rows to be between two values.                                                                                    | Yes |
-| Volume             | `expect_table_row_count_to_equal`                         | Expect the number of rows to equal a value.                                                                                            | No |
-| Volume             | `expect_table_row_count_to_equal_other_table`             | Expect the number of rows to equal the number in another table within the same database.                                               | No |
+| Data quality issue |                    Expectation                       |                                                               Description                                                              | Dynamic Parameters? |
+|------------------|----------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------|-------------------|
+| Completeness       | **column values to be null**                         | Expect the column values to be null.                                                                                                   | Coming soon         |
```

Comment:
**note for reviewers:** Though not in the plan, I updated the Expectation names to remove "Expect" from the beginning of each one to match how these options will be presented in the UI. I also styled them in bold as references to UI options.